rbi/core/object.rbi

```diff
@@ -5,4 +5,9 @@ class Object < BasicObject
 
   sig {returns(Integer)}
   def object_id(); end
+
+  sig { params(blk: T.proc.params(arg: T.untyped).returns(T.untyped)).returns(T.untyped) }
```

Comment:
```suggestion
  sig {params(blk: T.proc.params(arg: T.untyped).returns(T.untyped)).returns(T.untyped)}
```

to be consistent with Stripe's Ruby style

Reply:
Thank you :-)